ci/copy-windows-release-artifacts.sh

```diff
@@ -16,12 +16,13 @@ chmod +wx "$INSTALLER"
 chmod +wx "$EE_INSTALLER"
 
 if ! [ -f /C/Users/u/.dotnet/tools/azuresigntool.exe ]; then
-    "/C/Program Files/dotnet/dotnet.exe" tool install --global AzureSignTool
+    "/C/Program Files/dotnet/dotnet.exe" tool install --global AzureSignTool --version 3.0.0
```

Comment:
we need to backport this to the 1.7.0 release branch

Reply:
https://github.com/digital-asset/daml/pull/10994

Reply:
Oh, right. Thanks!